lib/cdo/delete_accounts_helper.rb

```diff
@@ -110,9 +110,12 @@ def clean_and_destroy_pd_content(user_id)
     Pd::FacilitatorProgramRegistration.where(user_id: user_id).update_all(form_data: '{}')
     Pd::RegionalPartnerProgramRegistration.where(user_id: user_id).update_all(form_data: '{}', teachercon: 0)
     Pd::Teachercon1819Registration.where(user_id: user_id).update_all(form_data: '{}', user_id: nil)
-    Pd::TeacherApplication.where(user_id: user_id).update_all(primary_email: '', secondary_email: '', application: '')
     Pd::RegionalPartnerContact.where(user_id: user_id).update_all(form_data: '{}')
 
+    # SQL query to anonymize Pd::TeacherApplication (2017-18 application) because the model no longer exists
+    ActiveRecord::Base.connection.exec_query(
+      "UPDATE `pd_teacher_applications` SET `pd_teacher_applications`.`primary_email` = '', `pd_teacher_applications`.`secondary_email` = '', `pd_teacher_applications`.`application` = '' WHERE `pd_teacher_applications`.`user_id` = #{user_id}"
+    )
```

Comment:
Small style nit: This is a long line.  We could use heredoc to split the query over multiple lines for readability.